### PR TITLE
KVM tenant cluster release 13.1.0 with QEMU 5.2.0

### PR DESCRIPTION
**Pull request by tfussell**
Towards https://github.com/giantswarm/roadmap/issues/262

### CODE REVIEW

File: kvm/v13.1.0/release.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    releaseOperatorDeploy: true
+    version: 0.23.19
+  - name: containerlinux
+    version: 2605.10.0
+  - name: etcd
+    version: 3.4.13
+  - name: flannel-operator
+    version: 0.2.0
+  - name: kubernetes
+    version: 1.18.14
```
Comment (stone-z) [inline kvm/v13.1.0/release.yaml:46]: 13.0.0 was `1.18.12` -- if this bump is intentional then it should have notes as well

Comment (tfussell) [reply kvm/v13.1.0/release.yaml:46]: Good call. I'll add it.

File: kvm/v13.1.0/release.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    releaseOperatorDeploy: true
+  - name: calico
+    version: 3.15.3
+  - name: cert-operator
+    releaseOperatorDeploy: true
+    version: 0.1.0
+    reference: 0.1.0-2
+  - name: cluster-operator
+    releaseOperatorDeploy: true
+    version: 0.23.19
```
Comment (stone-z) [inline kvm/v13.1.0/release.yaml:38]:
```suggestion
    version: 0.24.0
```

File: kvm/v13.1.0/release.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  components:
+  - name: app-operator
+    version: 2.7.0
+    releaseOperatorDeploy: true
+  - name: calico
+    version: 3.15.3
+  - name: cert-operator
+    releaseOperatorDeploy: true
+    version: 0.1.0
+    reference: 0.1.0-2
```
Comment (stone-z) [inline kvm/v13.1.0/release.yaml:35]:
```suggestion
    version: 1.0.0
```

**Comment by tfussell** [comment]
Closing in favor of https://github.com/giantswarm/releases/pull/626

**Pull request closed without merging**